CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 59.5.0
 
 * Adds `combine_mode` parameter to Email Alert API `find_subscriber_list` method
```

Comment:
We've missed it, but there should be a `.` here.

Reply:
Some releases have one, some don't.  It's all very inconsistent 😞.